cmd/client/grpc_client.go

```diff
@@ -23,12 +25,38 @@ const (
 	FlagReadRemote  = "read-remote"
 	FlagWriteRemote = "write-remote"
 
+	FlagInsecureNoTransportSecurity  = "insecure-disable-transport-security"
+	FlagInsecureSkipHostVerification = "insecure-skip-hostname-verification"
+
 	EnvReadRemote  = "KETO_READ_REMOTE"
 	EnvWriteRemote = "KETO_WRITE_REMOTE"
+	EnvAuthToken   = "ORY_PAT"
```

Comment:
I would use a more generic name, as we might also rename the tokens in cloud and this is Ory cloud specific. What about `KETO_BEARER_TOKEN`?

Reply:
:+1: I took `ORY_PAT` specifically because it is used in Cloud.